homeassistant/components/homematicip_cloud/alarm_control_panel.py

```diff
@@ -44,6 +52,10 @@ def __init__(self, home: AsyncHome, device) -> None:
         device.modelType = 'Group-SecurityZone'
         device.windowState = None
         super().__init__(home, device)
+        _LOGGER.error("Homematic IP: %s %s is deprecated. Please switch to "
```

Comment:
Please move the logging outside init to just under line 38. We don't want side effects in init method.

Reply:
Make it a warning.